python/taichi/math/mathimpl.py

```diff
@@ -548,40 +588,41 @@ def rot2(ang):
         >>> from taichi.math import *
         >>> @ti.kernel
         >>> def test():
-        >>>     M = rot2(radians(30))
+        >>>     M = rotation2d(radians(30))
         [[0.866025, -0.500000], [0.500000, 0.866025]]
     """
     ca, sa = ti.cos(ang), ti.sin(ang)
     return mat2([[ca, -sa], [sa, ca]])
 
 
 @ti.func
-def rot3(axis, ang):
-    """Returns the matrix representation of a 3d rotation,
-    given the axis and angle of rotation.
+def rotation3d(ang_x, ang_y, ang_z):
+    """ Creates a 3D 4 * 4 homogeneous rotation matrix from an euler angle(Y * X * Z).
```

Comment:
The same as above, we should explain more on the axis and the order of rotations.